sdk/ml/azure-ai-ml/pyproject.toml

```diff
@@ -28,7 +28,31 @@ extend_skip_glob = [
 ]
 
 [tool.mypy] 
-exclude = ["azure/ai/ml/_vendor", "azure/ai/ml/_restclient"] 
+exclude = ["azure/ai/ml/_vendor",
+"azure/ai/ml/_restclient",
+"azure/ai/ml/_schema",
+"azure/ai/ml/_internal",
+"azure/ai/ml/exceptions.py",
+"azure/ai/ml/_azure_environments.py",
+"azure/ai/ml/_scope_dependent_operations.py",
+"azure/ai/ml/_ml_client.py",
+"azure/ai/ml/_exception_helper.py",
+"azure/ai/ml/_sweep",
+"azure/ai/ml/_parallel",
+"azure/ai/ml/operations",
```

Comment:
Make sure you unexclude the ones we've merged since like operations

Reply:
Sure